nilearn/plotting/html_surface.py

```diff
@@ -127,8 +127,8 @@ def _fill_html_template(info, embed_js=True):
 
 def view_img_on_surf(stat_map_img, surf_mesh='fsaverage5',
                      threshold=None, cmap=cm.cold_hot,
-                     black_bg=False, vmax=None, vmin=None,
-                     symmetric_cmap=True):
+                     black_bg=False, vmax=None, vmin=None, symmetric_cmap=True,
+                     colorbar=True, cbar_height=.5, cbar_fontsize=25):
```

Comment:
We are calling it `colorbar` and also `cbar`.
We should call it `colorbar`, `colorbar_height`, `colorbar_fontsize`.
Saving a few extra characters isn't worth the clarity, especially when most users will rely on autocomplete.